game/scripts/npc/items/item_satanic_core_3.txt

```diff
@@ -98,7 +98,7 @@
 			"10"
 			{
 				"var_type"				"FIELD_FLOAT"
-				"hero_lifesteal"		"50 65 80"
```

Comment:
I have my own review of Satanic Core pending. Would you be willing to let me have that? Or at least give it a look?